elements/bulbs-truncate-element/bulbs-truncate-element.scss

```diff
@@ -0,0 +1,27 @@
+.bulbs-truncate-element-parent-active {
```

Comment:
Leaving off the animation because it's not quick and easy to animate between `max-height:1000px` and `max-height:auto`. We can come back to this after QA has a look at the functionality.